PURL-TYPES.rst

```diff
@@ -23,6 +23,23 @@ See also https://github.com/package-url/purl-spec and
 Known ``purl`` types
 ~~~~~~~~~~~~~~~~~~~~
 
+apk
+---
+``apk`` for APK-based packages:
+
+- There is no default package repository: this should be implied either from
+  the ``distro`` qualifiers key  or using a repository base url as 
+  ``repository_url`` qualifiers key.
+- The ``namespace`` is the vendor such as ``alpine`` or ``openwrt``.
```

Comment:
If we compare the namespace of the "rpm" purl type, rpm states that "It is not case sensitive and must be lowercased.". What is the criteria for the apk namespace?

Reply:
It's largely similar to how I structured the spec for ALPM. I somehow forgot the line so I've added it back into the section. I'm not aware of any other namespace criterias.